numpy/core/src/multiarray/einsum_sumprod.c.src

```diff
@@ -501,19 +501,19 @@ static NPY_GCC_OPT_3 void
             npyv_@sfx@ a@i@ = npyv_@ld@_@sfx@(data0 + vstep * @i@);
             npyv_@sfx@ b@i@ = npyv_@ld@_@sfx@(data1 + vstep * @i@);
             /**end repeat3**/
-            npyv_@sfx@ ab3 = npyv_muladd_@sfx@(a3, b3, vaccum);
+            npyv_@sfx@ ab3 = npyv_muladd_@sfx@(a3, b3, vacuum);
             npyv_@sfx@ ab2 = npyv_muladd_@sfx@(a2, b2, ab3);
             npyv_@sfx@ ab1 = npyv_muladd_@sfx@(a1, b1, ab2);
-                    vaccum = npyv_muladd_@sfx@(a0, b0, ab1);
+                    vacuum = npyv_muladd_@sfx@(a0, b0, ab1);
         }
     }
     /**end repeat2**/
     for (; count > 0; count -= vstep, data0 += vstep, data1 += vstep) {
         npyv_@sfx@ a = npyv_load_tillz_@sfx@(data0, count);
         npyv_@sfx@ b = npyv_load_tillz_@sfx@(data1, count);
-        vaccum = npyv_muladd_@sfx@(a, b, vaccum);
```

Comment:
This fix is wrong.  The work is not vacuum. It is essentially v_accum.

Reply:
Ah, sorry, I thought it was some sort of joke because of the initialization to 0's, or something like that. Will revert it.